test/IECoreImage/ImageWriterTest.py

```diff
@@ -835,6 +835,7 @@ def tearDown( self ) :
 			"test/IECoreImage/data/jpg/output.jpg",
 			"test/IECoreImage/data/dpx/output.dpx",
 			"test/IECoreImage/data/tiff/output.tif",
+			"test/IECoreImage/data/png/output.png",
```

Comment:
Until this line was added, `setUp()` was removing all the same files as `tearDown()`, so the two functions no longer match. I'm not sure what the logic behind that was though. Presumably we want to avoid false successes caused by files already existing from a previous run, but I'm not sure why `setUp()` can't trust `tearDown()` to have done that. Maybe because of a hard crash in a previous run? The better approach would probably just to use `temporaryDirectory()` so we have a unique output location per run.

Reply:
I force rebased into a893e0d, removing `setUp` and adding a todo in `tearDown` to use `temporaryDirectory()` in the future.